README.md

```diff
@@ -1,92 +1,150 @@
+    - Install `nasm-<VERSION>-installer-x64.exe`
+        - Run as normal user and click OK 
+        - Install in default directory `C:\Users\<USER>\AppData\Local\bin\NASM`
+- **Recommended: When creating NSIS Windows installer**:
+    - Install NSIS: `nsis-<NSIS_VERSION>-setup.exe`
+    - Extract NSIS `LockedList.zip` plugin:
+        - Copy `LockedList\Plugins\LockedList64.dll` to `C:\Program Files (x86)\NSIS\Plugins\LockedList64.dll`
+        - Copy `LockedList\Plugins\x86-unicode\LockedList.dll` to `C:\Program Files (x86)\NSIS\Plugins\x86-unicode\LockedList.dll`
+    - Extract NSIS `Inetc.zip` plugin:
+        - Copy `Inetc\Plugins\x86-unicode\INetC.dll` to `C:\Program Files (x86)\NSIS\Plugins\x86-unicode\INetC.dll`
+    - Extract NSIS `Registry.zip` plugin:
+        - Copy `Registry\Desktop\Plugin\registry.dll` to `C:\Program Files (x86)\NSIS\Plugins\registry.dll`
+        - Copy `Registry\Desktop\Plugin\registry.dll` to `C:\Program Files (x86)\NSIS\Plugins\x86-unicode\registry.dll`
+
+Delete `pkg-config` files to prevent conflicts with Strawberry's own `pkg-config`:
+- Delete `C:\strawberry\perl\bin\pkg-config`
+- Delete `C:\strawberry\perl\bin\pkg-config.bat`
+
+Windows Settings | System | About | Advanced system settings | Tab Advanced | Environment Variables:
+- System variables | Path | Edit:
+    - Delete: `C:\Strawberry\c\bin` (To prevent conflicts with other utilities)
+    - Add: `C:\Program Files\Git\bin` (This is for `sed` and `patch` utilities)
+    - Add: `C:\Users\user\AppData\Local\bin\NASM`
+    - Add: `C:\data\projects\strawberry\msvc_\downloads\win_flex_bison-<VERSION>`
+    - Add: `C:\Program Files (x86)\NSIS`
+    - Make sure no MinGW-W64 (gcc, g++, etc) installation is in PATH, as this can cause those to be picked up as compiler instead of MSVC.
+
+### Meson installation
+
+Building with Meson 1.6.0 installation results in build issues
+[meson: error: unrecognized arguments:](https://github.com/strawberrymusicplayer/strawberry-msvc/issues/6).
+
+A temporary solution is to remove Meson via Windows Settings | `Apps or remove programs`.
+Then install Meson via Python PIP system wide. Start | `x64 Native Tools Command Prompt for VS 2022` and type:
+
+```
+pip install meson
+```
+
+## Optional: Prebuilt binary dependencies
+
+Prebuilt MSVC binaries can be optionally used to speed-up the build process. When this step is skipped, all libraries and dependencies are build from source.
+
+To use prebuilds, download the following `tar.xz` files from Github [strawberry-msvc-dependencies/releases](https://github.com/strawberrymusicplayer/strawberry-msvc-dependencies/releases):
+- For debug, extract `strawberry-msvc-x86_64-debug.tar.xz` to `C:\strawberry_msvc_x86_64_debug`
+- For release, extract `strawberry-msvc-x86_64-release.tar.xz` to `C:\strawberry_msvc_x86_64_release`
+
+## Build Strawberry from source including dependencies
+
+- Set one debug or release in `Path` environment variable:
```

Comment:
`PATH `not `Path`

Reply:
On Windows 11 this changed to `Path`:

![image](https://github.com/user-attachments/assets/199e96c5-e041-4751-b37e-7052d3af0754)

Both `set Path` and `set PATH` works as it is not case sensitive like Linux.